config/manager/deployment.yaml

```diff
@@ -52,9 +52,6 @@ spec:
         imagePullPolicy: Always
         name: manager
         resources:
-          limits:
-            cpu: 100m
-            memory: 500Mi
```

Comment:
Should the memory limit go as well? His request seemed specific to CPU but I don't know if you can even have a memory limit and no cpu specified.

Reply:
The test checks for the `memory` attribute the same way it does the `cpu`, so it seems it needs to go as well.

Reply:
Very good catch, thanks.

Reply:
i am less concerned about a memory limit than cpu limits.  The test only fails on cpu limits.